### PR TITLE
refactor: add Hex type to protocol and restrain all hex data started with 0x

### PR DESCRIPTION
**Pull request by mkxbl**
**What type of PR is this?**

refactor: hex data type and restrain 0x format

**What this PR does / why we need it**:

1. api: restrain hex data started with 0x, both hex data in tx and service payload

2. protocol: add `struct Hex(String)`, and hex string should use `Hex` not `String`. 
Modify some String type to Hex type, eg: Config(privkey)、ConfigNetworkBootstrap(pubkey)、Genesis(prehash)、Metadata(common_ref, bls_pub_key)

3. chore: update all config file

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by yejiayu** [comment]
The toolchain needs to be updated, such as the SDK and benchmark.

@homura

File: core/api/src/lib.rs [hunk 1]
```diff
@@ -59,7 +59,7 @@ impl Query {
 
     #[graphql(name = "getTransaction", description = "Get the transaction by hash")]
     async fn get_transaction(state_ctx: &State, tx_hash: Hash) -> FieldResult<SignedTransaction> {
-        let hash = protocol::types::Hash::from_hex(&tx_hash.as_hex())?;
+        let hash = protocol::types::Hash::from_hex(&tx_hash.as_hex()?)?;
```
Comment (homura) [inline core/api/src/lib.rs]: The input forces 0x, so it is recommended that output also starts with 0x.
It is recommended to modify the output hex of the `from_hex` method to start with 0x

Comment (mkxbl) [reply core/api/src/lib.rs]: done

Comment (mkxbl) [reply core/api/src/lib.rs]: please check again

**Comment by mkxbl** [comment]
add `pub struct Hex(String)` to protocol types for some hex string. 

And modify some String type to Hex type, eg: Config(privkey)、ConfigNetworkBootstrap(pubkey)、Genesis(prehash)、Metadata(common_ref, bls_pub_key)

**Comment by yejiayu** [comment]
@mkxbl Modify the keypairs tool(devtools/keypairs)

**Comment by mkxbl** [comment]
> @mkxbl Modify the keypairs tool(devtools/keypairs)

done

**Pull request merged**